cl/donate/api_views.py

```diff
@@ -231,6 +236,24 @@ def _handle_membership_creation_or_update(self, webhook_data) -> None:
         if membership_data["status"] not in ["succeeded", "succeed"]:
             return None
         user = self._get_member_record(membership_data["accountId"])
+        membership_level = NeonMembership.TYPES_INVERTED[
+            membership_data["membershipName"]
+        ]
+
+        if membership_level == NeonMembership.EDU:
+            is_valid_edu_account = (
+                user.profile.email_confirmed and user.email.endswith(".edu")
```

Comment:
I don't think we need to deal with this now, but we do have an increasing number of folks from other countries signing up with their academic email addresses. Eventually we'll want to deal with them, but we can see how much they complain about this (if at all).